app/javascript/onboarding/components/FollowUsers.jsx

```diff
@@ -175,42 +181,51 @@ export class FollowUsers extends Component {
               </div>
             </header>
 
-            <div data-testid="onboarding-users">
-              {users.map((user) => (
-                <button
-                  data-testid="onboarding-user-button"
-                  type="button"
-                  onClick={() => this.handleClick(user)}
-                  onKeyDown={() => this.handleKeyDown(user)}
-                  className={
-                    selectedUsers.includes(user)
-                      ? 'user content-row selected'
-                      : 'user content-row unselected'
-                  }
-                >
-                  <figure className="user-avatar-container">
-                    <img
-                      className="user-avatar"
-                      src={user.profile_image_url}
-                      alt="profile"
-                    />
-                  </figure>
-                  <div className="user-info">
-                    <h4 className="user-name">{user.name}</h4>
-                    <p className="user-summary">
-                      {he.unescape(user.summary || '')}
-                    </p>
-                  </div>
-                  <button
-                    data-testid="onboarding-user-following-status"
-                    type="button"
-                    className="user-following-status"
+            <fieldset data-testid="onboarding-users">
+              {users.map((user) => {
+                const selected = selectedUsers.includes(user);
+
+                return (
+                  <div
+                    data-testid="onboarding-user-button"
+                    className={`user content-row ${
+                      selected ? 'selected' : 'unselected'
+                    }`}
                   >
-                    {selectedUsers.includes(user) ? 'Following' : 'Follow'}
-                  </button>
-                </button>
-              ))}
-            </div>
+                    <figure className="user-avatar-container">
+                      <img
+                        className="user-avatar"
+                        src={user.profile_image_url}
+                        alt="profile"
```

Comment:
```suggestion
                        alt=""
```

small change (blocking) - While we're cleaning up accessibility issues here, let's remove this alt text - it's redundant since the image is decorative, and even if not, the text "profile" isn't helpful

Reply:
I have made the changes @aitchiss